### PR TITLE
[incubator/fluentd] Make ES buffer params configurable

### PR DESCRIPTION
**Pull request by coreypobrien**

### CODE REVIEW

File: incubator/fluentd/values.yaml [hunk 1]
```diff
@@ -12,6 +12,12 @@ elasticsearch:
   port: 443
   scheme: https
   extraArgs: []
+  # This is set low to accomodate m4.large ES clusters that have a 10m client body limit
+  buffer_chunk_limit: 5m
```
Comment (dosullivan) [inline incubator/fluentd/values.yaml:16]: maybe we could make a map in a future release to provide multiple defaults for `buffer_chunk_limit` and `buffer_queue_limit` based on instance type?

Comment (coreypobrien) [reply incubator/fluentd/values.yaml:16]: Definitely 👍These are targeted toward minimums, but could be way more efficient if instance allowed for it.

**Review by robscott — APPROVED** [review]
Looks good to me, thanks!

**Review by davekonopka — APPROVED** [review]
👍🏻

**Pull request merged**